source/docs/contracts_configuration.md

```diff
@@ -311,11 +318,26 @@ contracts: {
   },
 },
 afterDeploy: (dependencies) => {
-  dependencies.contracts.SimpleStorage.methods.set(150).send();
+  dependencies.contracts.SimpleStorage.methods.set(150).send({from: dependencies.web3.eth.defaultAccount});
 }
 ...
 </code></pre>
 
+### Error management
+
+Since we use functions for these deployment hooks instead of the vintage strings, we have to manage errors ourselves.
```

Comment:
At this point, as a new user I'd be wondering what you mean by "vintage strings". 

I think we should write the docs always in a way that they reflect the latest and greatest and mention explicitly when something was different before. Since error handling wasn't really a think within strings, it might not even be needed to mention "vintage strings" in the first place.

Reply:
> since error handling wasn't really a think within strings

Just an FYI, but Embark did do some error handling on the strings in the fact that it caught any error thrown during the eval and outputed it to the user, that's why I talk about the vintage strings, but I agree that we don't need to specify it